keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResourceHandler.kt

```diff
@@ -155,11 +128,9 @@ abstract class ResourceHandler<S : ResourceSpec, R : Any>(
   open suspend fun delete(resource: Resource<S>): List<Task> = TODO("Not implemented")
 
   /**
-   * Generate a Keel SubmittedResource definition from currently existing resources.
+   * Generate a spec from currently existing resources.
    */
-  open suspend fun export(
-    exportable: Exportable
-  ): SubmittedResource<*> {
+  open suspend fun export(exportable: Exportable): S {
```

Comment:
Made `export` just return the spec as it wasn't doing anything interesting with `SubmittedResource` (all implementations were identical. I'm hoping to keep that class out of `keel-api`.